logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="logistics_schedule_view_tree" model="ir.ui.view">
+        <field name="name">logistics.schedule.tree</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <field name="partner_id" />
+                <field name="stock_move_id" domain="[('partner_id', '=', partner_id), ('product_id', '=', product_id), ('state', '=', 'done'), ('logistics_schedule_id', '=', False)]"/>
+                <field name="transport_type" />
+                <field name="product_id" />
+                <field name="product_uom_qty" />
+                <field name="scheduled_load_date" />
+                <field name="commitment_date" />
+                <field name="logistic_price_unit"/>
+                <field name="logistic_price_unit_done" />
+                <field name="carrier_id" />
+                <field name="license_plate_1" />
+                <field name="license_plate_2" />
+                <field name="state" readonly="1"/>
+                <field name="note" />
+                <button
+                    name="action_logistics_schedule_form_view"
+                    icon="fa-gear"
+                    class="p-0"
+                    type="object"
```

Comment:
Is it possible to create an XML action window and call it instead of forcing a `type=object` button?